common/src/View/InputEvent.cpp

```diff
@@ -63,7 +63,7 @@ namespace TrenchBroom {
             return out;
         }
         
-        MouseEvent::MouseEvent(const Type i_type, const Button i_button, const WheelAxis i_wheelAxis, const int i_posX, const int i_posY, const float i_scrollDistance):
+        MouseEvent::MouseEvent(const Type i_type, const Button i_button, const WheelAxis i_wheelAxis, const double i_posX, const double i_posY, const float i_scrollDistance):
```

Comment:
float is more than enough precision for these (1 ULP above 4096 is 4096.00048828 for a float32), so you should use whichever of float/double is more convenient.

Wouldn't float be more convenient? I see some `static_cast<float>`'s added in this PR when generating pick rays

Reply:
Changed to `float` . I initially used double because that's what Qt provides. But float makes more sense for us.